packages/story-editor/src/components/mediaRecording/footer.js

```diff
@@ -287,10 +328,15 @@ function Footer({ captureImage, videoRef }) {
           <RetryButton onClick={onRetry} disabled={isInserting}>
             {__('Retry', 'web-stories')}
           </RetryButton>
-          <InsertButton onClick={onInsert} disabled={isInserting}>
+          <InsertButton
+            onClick={hasVideo ? onInsert : onAudioInsert}
+            disabled={isInserting}
+          >
             {isInserting
               ? __('Inserting…', 'web-stories')
-              : __('Insert', 'web-stories')}
+              : hasVideo
+              ? __('Insert', 'web-stories')
+              : __('Insert background audio', 'web-stories')}
```

Comment:
So it should be 
`Set background audio` and `Settings background audio`?

Reply:
Current wording is OK for now